lib/MusicBrainz/Server/Form/Work.pm

```diff
@@ -145,11 +146,23 @@ after 'validate' => sub {
             next;
         }
 
+        my @used_attribute_values = @{ $used_attributes{$attribute_type} // [] };
```

Comment:
Try `//= []`, then you can remove the line `$used_attributes{$attribute_type} = \@used_attribute_values;` below.